cluster/resolver/rpc.go

```diff
@@ -9,32 +9,34 @@
 //  CONTACT: hello@weaviate.io
 //
 
-package rpc
+package resolver
 
 import (
 	"fmt"
 	"net"
 	"strconv"
 )
 
-// NewResolver returns an implementation of rpcAddressResolver
-// isLocalHost is used to determine which remote port to expect given an address (See: rpcResolver.rpcAddressFromRAFT())
-// rpcPort is used as the default port on the returned rpcAddresses (see: rpcResolver.Address())
-func NewResolver(isLocalHost bool, rpcPort int) rpcAddressResolver {
-	return &rpcResolver{isLocalCluster: isLocalHost, rpcPort: rpcPort}
-}
-
-type rpcResolver struct {
+// RpcResolver implements resolving raft address to their RPC address depending on the configured rpcPort and whether or
+// not this is a local cluster.
+type RpcResolver struct {
```

Comment:
**thought (non-blocking):**  shall we call it `RPC` only given that it would be used `resolver.RPC`.  wdyt?